src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/DotnetDataCollectionLauncher.cs

```diff
@@ -118,6 +118,8 @@ public override int LaunchDataCollector(IDictionary<string, string> environmentV
             var cliArgs = string.Join(" ", commandLineArguments);
             var argumentsString = string.Format("{0} \"{1}\" {2} ", args, dataCollectorAssemblyPath, cliArgs);
 
+            currentWorkingDirectory = Directory.GetCurrentDirectory();
```

Comment:
Please create another variable for this purpose.
Fix it it DefaultDataCollectionLauncher`